improver_tests/developer_tools/test_MOMetadataInterpreter.py

```diff
@@ -0,0 +1,409 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2021 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Unit tests for the MOMetadataInterpreter plugin
+
+Description of imports: fixture name -> cube name:
+- ensemble_fixture -> ensemble_cube
+- landmask_fixture -> landmask_cube
+- percentile_fixture -> wind_gust_percentile_cube
+- probability_above_fixture -> probability_above_cube
+- probability_below_fixture -> blended_probability_below_cube
+- snow_level_fixture -> snow_level_cube
+- spot_fixture -> blended_spot_median_cube
+- wind_direction_fixture -> wind_direction_cube
+- wxcode_fixture -> wxcode_cube
+"""
+
+import numpy as np
+import pytest
+from iris.coords import CellMethod
+
+from improver_tests.developer_tools import (
+    ensemble_fixture,
```

Comment:
Suggest removing these imports and the docstring explanation of them above - see comment above about `conftest.py`.